tests/test_vectors.py

```diff
@@ -315,8 +315,6 @@ def test_list_vectors():
     vector_set = ns.vectors()
 
     assert len(vector_set) == 98
-    assert vector_set[0].id == 7
-    assert vector_set[0].vector == [0.7, 0.7]
```

Comment:
we don't care about ordering, but we still maybe we want to make sure we get this vector? could we try to find its position and assert its there instead?